### ISSUE TITLE
Bug: Barcodes are not deleted when deleting a product

### ISSUE THREAD

**Issue by tonytonyistony** [grocy/grocy#1926]
I have ran into issues when deleting products that when I try to re-input the item that it says the barcode already exists. Would be nice to have a section in the GUI of all your barcodes which reside in the DB file.

**Comment by berrnd** [comment]
> Would be nice to have a section in the GUI of all your barcodes which reside in the DB file.

More sense makes of course to delete all barcodes for that product when deleting it. That's currently missing and will be added.

**Comment by berrnd** [comment]
Turns out that this is the case since v3.3.0 (ref https://github.com/grocy/grocy/commit/3e4f2eaf5df9d1d37d092a6f36dc24f04fbf72e7).

I would recommend doing an update and please report problems only after verifing them on the current release (that's also mentioned on the issue template).

**Comment by berrnd** [comment]
> I will also try to replicate the error step by step and re-report

Well, without that, no single problem can be solved and that's why that's also mentioned on the issue template.

I did (on the [pre-release demo](https://demo-prerelease.grocy.info)):
- Added a new product with barcode `123`
- Deleted that product
- Added a new product with barcode `123`
  - => No problem
